test/controllers/users_controller_test.rb

```diff
@@ -185,6 +185,23 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
     end
   end
 
+  describe 'remove_all_patients method' do
```

Comment:
Can you add a test here to confirm that it doesn't destroy patients in db? Something like `assert_no_difference 'Patient.count' do patch remove_all_patients(@user), ...` should do the trick.